src/test/java/com/google/api/codegen/testdata/ruby/ruby_main_longrunning.baseline

```diff
@@ -184,8 +178,8 @@ module Google
           )
         end
         @operations_stub = Google::Gax::Grpc.create_stub(
-          service_path,
-          port,
+          SERVICE_ADDRESS,
```

Comment:
We should to change these from frozen constants to ordinary attributes so that users are still able to mutate them if needed.

Reply:
They are only used in the constructor. I believe the suggestion was to override in a subclass or mixin.

Reply:
Ah, my mistake. You're right. Ignore that comment.